Doc/library/doctest.rst

```diff
@@ -83,7 +83,10 @@ Here's a complete but small example module::
 
    if __name__ == "__main__":
        import doctest
-       doctest.testmod()
+       import sys
+       fail, _ = doctest.testmod()
```

Comment:
Maybe it would be easier to have a new argument in `testmod()`, e.g.:

```python
doctest.testmod(exit_on_error=True)
```

Reply:
Such a new feature would require a new Python version to be released. There's already one way to do it, even supported in Python 2.